docker-compose-dev.yaml

```diff
@@ -0,0 +1,39 @@
+# PY_VER=3.8 IMAGE=djbase DISTRO=alpine PHARUS_VERSION=$(cat pharus/version.py | tail -1 | awk -F\' '{print $2}') HOST_UID=$(id -u) docker-compose -f docker-compose-dev.yaml up
+#
+# Intended for normal development. Supports hot/live reloading.
+# Note: If requirements or Dockerfile change, will need to add --build flag to docker-compose.
+# Make sure to add an entry into your /etc/hosts file as `127.0.0.1 fakeservices.datajoint.io`
+# This serves as an alias for the domain to resolve locally.
+# With this config and the configuration below in NGINX, you should be able to verify it is
+# running properly with a `curl https://fakeservices.datajoint.io/api/version`.
+version: "2.4"
+x-net: &net
+  networks:
+      - main
+services:
+  pharus:
+    <<: *net
+    extends:
+      file: ./docker-compose-build.yaml
+      service: pharus
+    environment:
+      - FLASK_ENV=development # enables logging to console from Flask
+    volumes:
+      - ./pharus:/opt/conda/lib/python3.8/site-packages/pharus
+    command: pharus
+  fakeservices.datajoint.io:
+    <<: *net
+    image: raphaelguzman/nginx:v0.0.15
```

Comment:
Should this be hosted on a datajoint dockerhub verse our own dev username?

Reply:
@Synicix Good catch. Unfortunately, this one is tricky to solve properly... It is a repo I created a long while back that I wasn't sure if we would need it but now it has grown to be dependent on various projects. I'd like to move it now but it would have breaking changes for other projects. Let's use it for now and I can mirror it to `datajoint`/`vathes` public repo and give some time for us to migrate things over.

Reply:
I will file an issue for it then. Just as a concern